### PR TITLE
feat: monitor livy & livy-spark3 logs and metrics

### LINKED ISSUE #164: livy: configure exporter and logs for tdp_observability
Configure livy and livy-spark3 services for observability : 
- Install and configure jmx-exporter
- Format log layout as in tdp-collection

### PR DESCRIPTION
**Pull request by giem-git**
#### Which issue(s) this PR fixes

Fixes #164

#### Agreements

- [x] I hereby declare this contribution to be licensed under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0).
- [x] I hereby agree to grant [TOSIT](https://www.tosit.io/) a copyright license to use my contributions.

### CODE REVIEW

**Comment by kpgtek** [comment]
It seems that the meta playbooks are not up-to-date :
```
playbooks/meta/livy-spark3.yml
playbooks/meta/livy.yml
```

File: tdp_vars_defaults/livy-spark3/livy-spark3.yml [hunk 1]
```diff
@@ -60,6 +60,22 @@ livy_conf:
   livy.superusers: "knox,hue,livy"
   livy.server.access-control.enabled: "true"
```
Comment (kpgtek) [inline tdp_vars_defaults/livy-spark3/livy-spark3.yml:62]: Maybe add a commentary for jmx_exporter variables (as for livy.yml):
```# jmx exporter config```

Comment (giem-git) [reply tdp_vars_defaults/livy-spark3/livy-spark3.yml:62]: Comment added

**Comment by giem-git** [comment]
> It seems that the meta playbooks are not up-to-date:
> 
> ```
> playbooks/meta/livy-spark3.yml
> playbooks/meta/livy.yml
> ```
> 
> Order of imported playbooks inside are different if they are regenerated from `tdp playbooks` command line.
Updated meta playbooks for livy with tdp generated files

**Review by kpgtek — APPROVED** [review]

File: playbooks/meta/livy-spark3.yml [hunk 2]
```diff
@@ -2,13 +2,14 @@
 # SPDX-License-Identifier: Apache-2.0
 
 ---
-- ansible.builtin.import_playbook: ../livy-spark3_server_install.yml
-- ansible.builtin.import_playbook: ../livy-spark3_kerberos_install.yml
-- ansible.builtin.import_playbook: ../livy-spark3_ssl-tls_install.yml
+- import_playbook: ../livy-spark3_jmx-exporter_config.yml
```
Comment (rpignolet) [inline playbooks/meta/livy-spark3.yml:5]: Can you update to use FQCN ? I know that this was generated by `tdp playbooks`. We should update the `ŧdp` command.

Comment (giem-git) [reply playbooks/meta/livy-spark3.yml]: FQCN updated. 
Note that `ansible-lint` also has the following complains : 
~~~
name[play]: All plays should be named.  
~~~

**Review by rpignolet — APPROVED** [review]

**Pull request merged**